.gitignore

```diff
@@ -75,3 +75,4 @@ obj~
 # This is used for local paths to SpacetimeDB packages.
 /nuget.config
 /nuget.config.meta
+.idea/
```

Comment:
unrelated change: This ignores the jetbrains `.idea/` directory which is created when opening the project in rider.